src/coordination/kubernetes/Akka.Coordination.KubernetesApi/Models/LeaseCustomResource.cs

```diff
@@ -39,7 +39,7 @@ public LeaseSpec()
         public LeaseSpec(string? owner, DateTime time)
         {
             Owner = owner;
-            Time = (long) time.TimeOfDay.TotalMilliseconds;
+            Time = time.Ticks;
```

Comment:
Stores the timeout value to absolute date and time expiration value instead of the millisecond offset until expiration